update: added information (#1801)
mia-platform/documentation: docs/console/company-configuration/clusters-management/cluster-setup.mdx

  ### ClusterRole

>   Next, we must define the roles needed by the previously created service account. 

Reviewer: ```suggestion
  Next, make sure to define the roles needed by the previously created service account. 
```

        - {{NAME_OF_THE_SERVICE_ACCOUNT_USED_FOR_THE_DEPLOY}}
  ```

  #### Monitoring custom resources from the Console

  Following these guidelines for configuring custom resources. 

>   We need to make sure that the console has the correct permissions 
>   to display the information to users.

Reviewer: ```suggestion
  You need to make sure that the Console has the correct permissions 
  to display the information to the users.
```

  Here is an example to show how to give the correct permissions to custom resource definitions:
  
  ```yaml
  apiVersion: rbac.authorization.k8s.io/v1
  kind: ClusterRole
  # ...
  rules:
    # For each custom resource definitions we need to add:
    - apiGroups:
        - "<custom-resource-definitions.group>"
      resources:
        - "<custom-resource-definitions.names.plural>"
      verbs:
        - get
        - list
  ```

  After adding the necessary permissions follow this [guide](/console/design-your-projects/custom-resources/custom-resources.md) to see your custom resources inside the console

  ### ClusterRoleBindings

>   Finally, we **must** associate the roles with the service account using the `ClusterRoleBindings` that can be created using the following template:

Reviewer: ```suggestion
  Finally, make sure to associate the roles with the service account using the `ClusterRoleBindings` that can be created using the following template:
```

Reviewer: must feels like too much on a documentation, plus there is inconsistency between 'we' and 'you'. I'd use always 'you', this is a guide for the user
